/**:
  ros__parameters:
    rate: 30.0
    stop_hold_acceleration: -1.5
